composer.json

```diff
@@ -0,0 +1,25 @@
+{
+	"name": "cloudinary/cloudinary-image-management-and-manipulation-in-the-cloud-cdn",
+	"homepage": "https://wordpress.org/plugins/cloudinary-image-management-and-manipulation-in-the-cloud-cdn/",
+	"type": "wordpress-plugin",
+	"license": "GPL-2.0-or-later",
+	"require": {},
+	"require-dev": {
+		"wp-coding-standards/wpcs": "^2.3",
+		"automattic/vipwpcs": "^2.2",
+		"dealerdirect/phpcodesniffer-composer-installer": "^0.6.2"
+	},
+	"config": {
+		"platform": {
+			"php": "5.6.20"
+		}
+	},
+	"scripts": {
+		"lint": [
+			"phpcs"
+		],
+		"lint-fix": [
```

Comment:
Can we consider `fix` (without the lint-) or `format` here?

Reply:
@dugajean 

I updated the action name.